modules/utility/init.zsh

```diff
@@ -30,8 +30,15 @@ else
   # colors for ls and completion
   export LSCOLORS='exfxcxdxbxGxDxabagacad'
   export LS_COLORS='di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'
-
-  alias ls='ls -G'
+ 
+  # stock OpenBSD ls does not support colors at all, but colorls does. 
+  if [[ $OSTYPE == openbsd* ]]; then
```

Comment:
I've done a little bit of research on major BSDs.
OSX, FreeBSD and DragonflyBSD have ls -G.
OpenBSD's ls does not have -G.
NetBSD's ls is also colorless, but users have 3! ways of getting color. 

1. install colorls (same trick as for OpenBSD) 
2. install coreutils (which includes ls) 
3. install gnuls

At this point I'm not sure how to deal with all that variety of ls versions. Having conditionals for every os and ls version is ugly.
maybe make a check you are on freebsd/dragonflybsd/osx and append -G, and leave ls alone for the rest of OS and let users deal with colors? This should be fair.
$OSTYPE on dflyBSD matches dragonfly*